go.mod

```diff
@@ -28,7 +31,12 @@ require (
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.19.0
+	github.com/strangelove-ventures/interchaintest/v7 v7.0.0-00010101000000-000000000000
```

Comment:
IIRC, versions >= v8.4.0 have ICS support built in. We will need to switch to it once we move main to v50 (probably today).

https://github.com/strangelove-ventures/interchaintest/releases/tag/v8.5.0